container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2454,8 +2454,11 @@ configuration:
       secret-generator:
         roles: [configgin-role, secrets-role]
         cluster-roles: [nonprivileged]
-      tests:
-        roles: [configgin-role, test-role]
+      tests-brain:
+        roles: [configgin-role, test-role-brain]
+        cluster-roles: [nonprivileged, test-cluster-role]
```

Comment:
Got out-of-band confirmation that we want to get rid of `test-cluster-role` too.

Reply:
Ok. @f0rmiga please ping me after this is done, and I will recheck in my vagrant